experimental/framework/fluid-static/src/FluidStatic.ts

```diff
@@ -4,64 +4,129 @@
  */
 
 import { getContainer, IGetContainerService } from "@fluid-experimental/get-container";
-import { getObjectWithIdFromContainer } from "@fluidframework/aqueduct";
+import { DataObject, getObjectWithIdFromContainer } from "@fluidframework/aqueduct";
 import { IContainer } from "@fluidframework/container-definitions";
-import { IFluidDataStoreFactory, NamedFluidDataStoreRegistryEntry } from "@fluidframework/runtime-definitions";
-import { DOProviderContainerRuntimeFactory } from "./containerCode";
+import { NamedFluidDataStoreRegistryEntry } from "@fluidframework/runtime-definitions";
+import { DOProviderContainerRuntimeFactory, IFluidStaticDataObjectClass } from "./containerCode";
 
-export interface IFluidStaticDataObjectClass {
-    readonly factory: IFluidDataStoreFactory;
+export interface FluidContainerConfig {
+    id: string,
+    dataObjects: IFluidStaticDataObjectClass[],
+}
+
+export interface FluidCreateContainerConfig extends FluidContainerConfig {
```

Comment:
So do we have a ContainerConfig & a ServiceConfig object? That sounds good to me!

Reply:
ServiceConfig === IGetContainerService

Reply:
Picking a naming pattern may be good.

Reply:
I agree. Let's do this as a follow up change since it's touching another package. We should think about where these definitions will go.